openshift/backplane-cli
# Move all functionality for assume command into credentials/console

### What type of PR is this?
cleanup/feature

### What this PR does / Why we need it?
takes the logic from the new assume/token commands (which are not yet in use anywhere) and consolidates them into the existing commands. this will prevent SREs from having to change anything about their existing workflows.

### Which Jira/Github issue(s) does this PR fix?
[OSD-19436](https://issues.redhat.com//browse/OSD-19436)

### Pre-checks (if applicable)


### Steps to test locally
As a pre-requisite, run either `backplane cloud console` or `backplane cloud credentials` against an existing, non-ROSA cluster and make sure it still works as it always has. For the new flow:

1. Create a new ROSA STS cluster in stage. The "new flow" should be enabled in the Red Hat org in stage.
2. Once the cluster is created and has been set up with all support roles, use "backplane cloud console" or "backplane cloud credentials" against that cluster.
Note - at this point, it is not uncommon to see a message on the screen saying "Waiting for IAM policy changes to resolve..."
3. Verify that legitimate credentials or a console url are printed out.
